plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -1156,25 +1158,28 @@ func (c *PodAffinityChecker) satisfiesExistingPodsAntiAffinity(pod *v1.Pod, meta
 		// present in nodeInfo. Pods on other nodes pass the filter.
 		filteredPods, err := c.podLister.FilteredList(nodeInfo.Filter, labels.Everything())
 		if err != nil {
-			glog.Errorf("Failed to get all pods, %+v", err)
-			return false
+			format := "Failed to get all pods, %+v"
```

Comment:
I think it would be cleaner (and a bit more efficient) if we did:

errMessage := fmt.sprintf("Failed to get all pods, %+v", err)
glog.Error(errMessage)
return ErrExistingPodsAntiAffinityRulesNotMatch, errors.New(errMessage)

If you agree, please change it for the cases below as well.

Reply:
yes, that's more cleaner than what i have done. :) thanks. Will do.